wafer/talks/tests/fixtures.py

```diff
@@ -0,0 +1,22 @@
+from wafer.talks.models import Talk, TalkType
+from wafer.tests.utils import create_user
+
+
+def create_talk_type(name):
+    """Create a talk type"""
+    return TalkType.objects.create(name=name)
+
+
+def create_talk(title, status, username=None, user=None, talk_type=None):
+    if username:
+        user = create_user(username)
```

Comment:
Should we check if both user and username are None and give a more informative error message in that case?

(probably should check for both set as well)